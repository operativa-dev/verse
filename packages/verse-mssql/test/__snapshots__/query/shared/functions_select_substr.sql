-- Executing SQL: Parameters: []
select substring("t1"."Title", 3 + 1, 2147483647) as "c0"
from "Album" as "t1"

