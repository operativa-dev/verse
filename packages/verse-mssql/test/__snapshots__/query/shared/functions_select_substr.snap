[
  " Those About To Rock We Salute You",
  "ls to the Wall",
  "tless and Wild",
  " There Be Rock",
  " Ones",
  "ged Little Pill",
  "elift",
  "ner 25 Anos",
  "ys Metallica By Four Cellos",
  "ioslave",
  " Of Exile",
  "kBeat Soundtrack",
  " Best Of Billy Cobham",
  "ohol Fueled Brewtality Live! [Disc 1]",
  "ohol Fueled Brewtality Live! [Disc 2]",
  "ck Sabbath",
  "ck Sabbath Vol. 4 (Remaster)",
  "y Count",
  "mical Wedding",
  " Best Of Buddy Guy - The Millenium Collection",
  "nda Minha",
  "inho Remix Ao Vivo",
  "ha Historia",
  "ociberdelia",
  "Lama Ao Caos",
  "stico MTV [Live]",
  "ade Negra - Hits",
  "Pista",
  " Bahia 2001",
  " Sessions [Disc 1] [Live]",
  "go Fury",
  "naval 2001",
  "ll: Brazil (Disc 1)",
  "ll: Brazil (Disc 2)",
  "age Inc. (Disc 1)",
  "atest Hits II",
  "atest Kiss",
  "rt of the Night",
  "ernational Superhits",
  "o The Light",
  "s Momentos",
  "ha História",
  "III The Final Concerts [Disc 1]",
  "sical Graffiti [Disc 1]",
  "bas De Enredo 2001",
  "ernatural",
  " Best of Ed Motta",
  " Essential Miles Davis [Disc 1]",
  " Essential Miles Davis [Disc 2]",
  " Final Concerts (Disc 2)",
  "An' Atom",
  "ícius De Moraes - Sem Limite",
  "es do MPB",
  "onicle, Vol. 1",
  "onicle, Vol. 2",
  "sia Eller - Coleção Sem Limite [Disc 2]",
  "sia Eller - Sem Limite [Disc 1]",
  "e Taste The Band",
  "p Purple In Rock",
  "eball",
  "cking at Your Back Door: The Best Of Deep Purple in the 80's",
  "hine Head",
  "pendicular",
  "ves And Masters",
  "rmbringer",
  " Battle Rages On",
  "lt: Def Leppard's Greatest Hits",
  "break",
  "van Ao Vivo - Vol. 02",
  "van Ao Vivo - Vol. 1",
  "s Regina-Minha História",
  " Cream Of Clapton",
  "lugged",
  "um Of The Year",
  "el Dust",
  "g For A Day Fool For A Lifetime",
  " Real Thing",
  "xa Entrar",
  "Your Honor [Disc 1]",
  "Your Honor [Disc 2]",
  " By One",
  " Colour And The Shape",
  "Way: The Best Of Frank Sinatra [Disc 1]",
  "a De Funk",
  "Canções de Eu Tu Eles",
  "nta Gente Veio Ver (Live)",
  "nta Gente Veio ver--Bônus De Carnaval",
  "eless",
  "rican Idiot",
  "etite for Destruction",
  " Your Illusion I",
  " Your Illusion II",
  "e Moods",
  "atter of Life and Death",
  "eal Dead One",
  "eal Live One",
  "ve New World",
  "ce Of Death",
  "r Of The Dark",
  "n Maiden",
  "lers",
  "e After Death",
  "e At Donington 1992 (Disc 1)",
  "e At Donington 1992 (Disc 2)",
  "Prayer For The Dying",
  "ce Of Mind",
  "erslave",
  "k In Rio [CD1]",
  "k In Rio [CD2]",
  "enth Son of a Seventh Son",
  "ewhere in Time",
  " Number of The Beast",
  " X Factor",
  "tual XI",
  " Machine",
  "rgency On Planet Earth",
  "kronized",
  " Return Of The Space Cowboy",
  " Born",
  " You Experienced?",
  "fing with the Alien (Remastered)",
  "ge Ben Jor 25 Anos",
  "a Quest-1995",
  "ezinho",
  "ing After Midnight",
  "lugged [Live]",
  " Sessions [Disc 2] [Live]",
  "a",
  "ses Of The Holy",
  "Through The Out Door",
  "",
  " Zeppelin I",
  " Zeppelin II",
  " Zeppelin III",
  "sical Graffiti [Disc 2]",
  "sence",
  " Song Remains The Same (Disc 1)",
  " Song Remains The Same (Disc 2)",
  "empestadeTempestade Ou O Livro Dos Dias",
  "s Do Mesmo",
  "atest Hits",
  "u Santos - RCA 100 Anos De Música - Álbum 01",
  "u Santos - RCA 100 Anos De Música - Álbum 02",
  "placed Childhood",
  "ulhinho Bom",
  "k And Shall Find: More Of The Best (1963-1981)",
  " Best Of Men At Work",
  "ck Album",
  "age Inc. (Disc 2)",
  "l 'Em All",
  "d",
  "ter Of Puppets",
  "oad",
  "e The Lightning",
  " Anger",
  "And Justice For All",
  "es Ahead",
  "ton Nascimento Ao Vivo",
  "as",
  " Of Spades",
  "orou...",
  "ley Crue Greatest Hits",
  "m The Muddy Banks Of The Wishkah [Live]",
  "ermind",
  "positores",
  "dum",
  "stico MTV",
  "uivo II",
  "uivo Os Paralamas Do Sucesso",
  "k at the Moon (Remastered)",
  "zzard of Ozz",
  "ry of a Madman (Remastered)",
  "More Tears (Remastered)",
  "bute",
  "king Into Clarksdale",
  "ginal Soundtracks 1",
  " Beast Live",
  "e On Two Legs [Live]",
  "rl Jam",
  "t Act",
  "",
  "",
  "k Side Of The Moon",
  "Cães Ladram Mas A Caravana Não Pára",
  "atest Hits I",
  "s Of The World",
  " Of Time",
  "en",
  " Adventures In Hi-Fi",
  " Best Of R.E.M.: The IRS Years",
  "ta Básica",
  "l Seixas",
  "od Sugar Sex Magik",
  "The Way",
  "ifornication",
  "rospective I (1974-1980)",
  "tana - As Years Go By",
  "tana Live",
  "uinarama",
  "amba Poconé",
  "as 0: B-Sides and Rarities",
  "ten Apples: Greatest Hits",
  "ides",
  "ning Dance",
  "Step",
  "e",
  "merize",
  "97] Black Light Syndrome",
  "e [Disc 1]",
  "e [Disc 2]",
  " Singles",
  "ond Good And Evil",
  "e Cult: The Best Of The Cult (For Rockers, Ravers, Lovers & Sinners) [UK]",
  " Doors",
  " Police Greatest Hits",
  " Rocks, 1964-1971 (Disc 1)",
  "Security",
  "doo Lounge",
  "gents",
  "nsmission",
  "Generation - The Very Best Of The Who",
  "ie Sem Limite (Disc 1)",
  "ie Sem Limite (Disc 2)",
  "stico",
  "ume Dois",
  "tlestar Galactica: The Story So Far",
  "tlestar Galactica, Season 3",
  "oes, Season 1",
  "t, Season 3",
  "t, Season 1",
  "t, Season 2",
  "tung Baby",
  " That You Can't Leave Behind",
  "ides 1980-1990",
  " To Dismantle An Atomic Bomb",
  "",
  "tle And Hum",
  " Best Of 1980-1990",
  "",
  "ropa",
  "0 The Best Of - Volume Two [UK]",
  "er Down",
  " Best Of Van Halen, Vol. I",
  " Halen",
  " Halen III",
  "traband",
  "icius De Moraes",
  "Vivo [IMPORT]",
  " Office, Season 1",
  " Office, Season 2",
  " Office, Season 3",
  "Led-Ed",
  "tlestar Galactica (Classic), Season 1",
  "aman",
  "tant Karma: The Amnesty International Campaign to Save Darfur",
  "ak of the Devil",
  "h Century Masters - The Millennium Collection: The Best of Scorpions",
  "se of Pain",
  "io Brasil (O Som da Jovem Vanguarda) - Seleccao de Henrique Amaro",
  "e: B-Sides and Rarities",
  "T, Season 4",
  "et Songs",
  "o Ko",
  "lize",
  "ry Kind of Light",
  "s II",
  "lds",
  " Best of Beethoven",
  "ple of the Dog",
  "ry On",
  "elations",
  "rate Deum: Gregorian Chant from the Proper of the Mass",
  "egri: Miserere",
  "helbel: Canon & Gigue",
  "aldi: The Four Seasons",
  "h: Violin Concertos",
  "h: Goldberg Variations",
  "h: The Cello Suites",
  "del: The Messiah (Highlights)",
  " World of Classical Favourites",
  " Neville Marriner: A Celebration",
  "art: Wind Concertos",
  "dn: Symphonies 99 - 104",
  "thoven: Symhonies Nos. 5 & 6",
  "oprano Inspired",
  "at Opera Choruses",
  "ner: Favourite Overtures",
  "ré: Requiem, Ravel: Pavane & Others",
  "aikovsky: The Nutcracker",
  " Last Night of the Proms",
  "cini: Madama Butterfly - Highlights",
  "st: The Planets, Op. 32 & Vaughan Williams: Fantasies",
  "arotti's Opera Made Easy",
  "at Performances - Barber's Adagio and Other Romantic Favorites for Strings",
  "mina Burana",
  "opland Celebration, Vol. I",
  "h: Toccata & Fugue in D Minor",
  "kofiev: Symphony No.1",
  "eherazade",
  "h: The Brandenburg Concertos",
  "pin: Piano Concertos Nos. 1 & 2",
  "cagni: Cavalleria Rusticana",
  "elius: Finlandia",
  "thoven Piano Sonatas: Moonlight & Pastorale",
  "at Recordings of the Century - Mahler: Das Lied von der Erde",
  "ar: Cello Concerto & Vaughan Williams: Fantasias",
  "ms, John: The Chairman Dances",
  "aikovsky: 1812 Festival Overture, Op.49, Capriccio Italien & Beethoven: Wellington's Victory",
  "estrina: Missa Papae Marcelli & Allegri: Miserere",
  "kofiev: Romeo & Juliet",
  "auss: Waltzes",
  "lioz: Symphonie Fantastique",
  "et: Carmen Highlights",
  "lish Renaissance",
  "del: Music for the Royal Fireworks (Original Version 1749)",
  "eg: Peer Gynt Suites & Sibelius: Pelléas et Mélisande",
  "art Gala: Famous Arias",
  "IABIN: Vers la flamme",
  "ada: Music from the Courts of England and Spain",
  "art: Symphonies Nos. 40 & 41",
  "k to Black",
  "nk",
  "ried to Dust (Bonus Track Version)",
  "thoven: Symphony No. 6 'Pastoral' Etc.",
  "tok: Violin & Viola Concertos",
  "delssohn: A Midsummer Night's Dream",
  "h: Orchestral Suites Nos. 1 - 4",
  "rpentier: Divertissements, Airs & Concerts",
  "th American Getaway",
  "ecki: Symphony No. 3",
  "cell: The Fairy Queen",
  " Ultimate Relexation Album",
  "cell: Music for the Queen Mary",
  "ll: The Seven Deadly Sins",
  ". Bach: Chaconne, Suite in E Minor, Partita in E Major & Prelude, Fugue and Allegro",
  "kofiev: Symphony No.5 & Stravinksy: Le Sacre Du Printemps",
  "manowski: Piano Works, Vol. 1",
  "lsen: The Six Symphonies",
  "at Recordings of the Century: Paganini's 24 Caprices",
  "zt - 12 Études D'Execution Transcendante",
  "at Recordings of the Century - Shubert: Schwanengesang, 4 Lieder",
  "atelli: Concertos for Violin, Strings and Continuo, Vol. 3",
  "pighi:Pines of Rome",
  "ubert: The Late String Quartets & String Quintet (3 CD's)",
  "teverdi: L'Orfeo",
  "art: Chamber Music",
  "aanisqatsi (Soundtrack from the Motion Picture)",
]